she_codes_news/news/models.py

```diff
@@ -1,8 +1,35 @@
+from django.contrib.auth import get_user_model
+# from django.contrib.auth.models import User
 from django.db import models
 
+User = get_user_model()
+
+class Category(models.Model):
+    name = models.CharField(max_length=64, unique=True)
+
+    class Meta:
+        verbose_name_plural = "categories"
+
+    def __str__(self):
+        return self.name
 
 class NewsStory(models.Model):
+    class Meta:
+        ordering = ["-pub_date"]
```

Comment:
I usually suggest against putting default ordering rules on models, i prefer to do this explicitly on the query rather than on the model!